daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/BitFunctions.tdml

```diff
@@ -353,26 +353,26 @@
         <tdml:infoset>
```

Comment:
Our indentation standard is 2-spaces, not 4 as you have here. 

Could you re-indent the XML files as 2-space indentation (generally an IDE setting change and then resave the files)?

With that, you can then squash all these commits together into one commit, and force push that. We will then merge it. 

Thanks.